### DISCUSSION TITLE
JanusGraph visualization in Graphlytic

### DISCUSSION THREAD

**Discussion by demtec** [JanusGraph/janusgraph#3964] [Show and tell]
Hey JanusGraphers, we have added support for JanusGraph in [Graphlytic](https://graphlytic.biz/) - our customizable graph visualization and analytics platform. We would love to get some feedback and suggestions on what we can improve.

Graphlytic can be installed [on-prem for free](https://graphlytic.biz/server) (1 user) or in our cloud.

**Comment by hadoopmarc** [comment]
Cool, so this could be mentioned now on the janusgraph frontpage (as beta visualization solution)!

**Reply by demtec** [reply]
That would be perfect! It's been in beta for almost a year and it's production-ready now (we will remove the "beta" tag in the next release).

Btw. I have noticed that there's still Linkurious mentioned on the JanusGraph webpage but they dropped support for JanusGraph in 2021 - https://linkurious.com/blog/linkurious-enterprise-3-0/ (see a paragraph at the end of the article)

**Reply by FlorianHockmann** [reply]
I think we should then just wait for your next release. If you want, you can then of course just create a PR to add Graphlytic to the README and [to the homepage](https://github.com/JanusGraph/janusgraph.org).

Regarding Linkurious, our link already pointed to their Ogma tool which still supports JanusGraph from what I can see. We should just change the text of the link to make it more clear that we are talking about Ogma here and not Linkurious Enterprise: #3975, janusgraph/janusgraph.org#99.

**Comment by demtec** [comment]
Graphlytic 4.2 was released, here's the announcement - [What's New in Graphlytic 4.2](https://graphlytic.com/blog/whats-new-in-graphlytic-4-2)

I tried to create a PR to add Graphlytic to the README and to JanusGraph homepage but was unsuccessful (lack of skill). Could you please add the link https://graphlytic.com there?

**Reply by FlorianHockmann** [reply]
Great to hear about the release. I just created the PRs:

#4004
JanusGraph/janusgraph.org#100

**Reply by demtec** [reply]
Ok, thank you very much Florian.

**Reply by FlorianHockmann** [reply]
The changes are now rolled out. We can keep this discussion however open as you've originally asked for feedback on Graphlytic which users/contributors can still provide.

**Reply by demtec** [reply]
Great! Thanks, Florian. 
Any feedback is appreciated and can be sent also to info@graphlytic.com
